active_shipping.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
 
   s.add_dependency("measured", ">= 2.0")
-  s.add_dependency("activesupport", ">= 4.2", "< 6.2")
+  s.add_dependency("activesupport", ">= 4.2", "~> 8.0")
```

This would require Rails 8 which is not what you want necessarily. I would just get less specific on the version.

```suggestion
  s.add_dependency("activesupport", ">= 4.2")
```